src/app/day/day.component.ts

```diff
@@ -105,6 +107,13 @@ export class DayComponent implements OnInit, OnChanges, AfterViewInit {
                 this.projectTimeChange.emit({ name: time.name, minutes: time.minutes });
             }
         }
+
+        // If timeSet is true will not set onf holiday as hours are already allocated for that day.
+        if (this.onfDay && timeSet === false) {
+            this.holidayMins = 480;
```

Comment:
i'm wondering if this should be moved to the back end altogether. When ever the ONF day is setup, it could just fill in 480 for holiday for all users.